chia/server/start_wallet.py

```diff
@@ -112,6 +114,8 @@ async def async_main() -> int:
 
 def main() -> int:
     freeze_support()
+    start_task_instrumentation()
+    atexit.register(stop_task_instrumentation)
```

Comment:
this needs to be done conditionally, like with a command like flag. However, the wallet is started by the daemon, and it's not trivial to pass on that here. Any help would be appreciated

Reply:
A couple options I can think of:

- environment variables. I believe the daemon doesn't sanitize this, so stop the daemon, then do `CHIA_WALLET_INSTRUMENT=1 chia start wallet`
- magic file. Look for presence of `chia-wallet-instrument-magic` in the current directory. But I guess this presupposes that the `PWD` environment variable is passed through, so if this works, the first way would work

Reply:
Stop the daemon with `chia stop -d all`

Reply:
Config option?

Reply:
It has to happen before we start the asyncio run loop, and we load the config after that. I liked the idea of using an environment variable